src/guides/v2.3/extension-dev-guide/build/di-xml-file.md

```diff
@@ -267,6 +267,70 @@ This mapping is in `app/etc/di.xml`, so the object manager injects the `Magento\
 
 This mapping is in `app/code/Magento/Backend/etc/adminhtml/di.xml`, so the object manager injects the `Magento\Backend\Model\Url` implementation class wherever there is a request for the `Magento\Core\Model\UrlInterface` in the [admin](https://glossary.magento.com/admin) area.
 
+### Override method using Preference
+
+If you want to override a public or protected method from core class, you can utilize `preference` to achieve it.
```

Comment:
```suggestion
If you want to override a public or protected method from a core class, you can utilize the `preference` node from `di.xml` to achieve it.
```

Reply:
@rogyar Updated the content as per your suggestion. Thanks for the review.